SpatialGDK/Documentation/content/handover-between-server-workers.md

```diff
@@ -19,20 +17,17 @@ Note that server-worker authority over properties is different to server-worker
 ## How to facilitate Actor handover
 
 To facilitate an Actor’s property handover between server-workers, follow the instructions below:
-1.  In the Actor’s class, mark the property field with a `Handover` tag in the `UPROPERTY` macro, as shown in the example below.
+1.  If your property is defined in a native c++ class, mark the property field with a `Handover` tag in the `UPROPERTY` macro, as shown in the example below.
```

Comment:
Capital C for C++? Not vital, but would be nice to be consistent across docs. (Disclaimer: I don't know if upper or lower case is the norm for our docs... but in the wider world I think capitalisation is common)